### DISCUSSION TITLE
Remember Last Positions of Duplicates

### DISCUSSION THREAD

**Discussion by the753q** [sumatrapdfreader/sumatrapdf#4524] [Ideas]
Problem: I'm reading a book, and want to quickly check the Contents page.
I realize there's no 'duplicate' button, so I just open the pdf again. (2 tabs of same pdf opened)

However, after later reopening Sumatra, it only remembered the position of latest pdf instance, which was on the contents page.
Now i lost where i was before and both pdfs are on contents page.

Suggestion: Remember positions of each instance of same document.
+ add a duplicate option when you click on the document tab.

**Comment by GitHubRulesOK** [comment]
It should be working in Current Pre-Release as long as both tabs are in the last window that window session will keep 2 separated entries for one file. [https://www.sumatrapdfreader.org/prerelease](https://www.sumatrapdfreader.org/prerelease)

```
SessionData [
	[
		TabStates [
			[
				FilePath = C:\Users\K\Documents\New Document.pdf
				DisplayMode = continuous
				PageNo = 1
				Zoom = fit page
				Rotation = 0
				ScrollPos = -1 -1
				ShowToc = false
			]
			[
				FilePath = C:\Users\K\Documents\New Document.pdf
				DisplayMode = continuous
				PageNo = 3
				Zoom = fit page
				Rotation = 0
				ScrollPos = -1 -1
				ShowToc = false
			]
		]
		TabIndex = 3
		WindowState = 1
		WindowPos = 774 227 855 793
		SidebarDx = 204
	]
]
```

If there were 2  or 20 ! independent windows then only one session can be remembered
```
SessionData [
	[
		TabStates [
			[
				FilePath = C:\Users\K\Documents\New Document.pdf
				DisplayMode = continuous
				PageNo = 3
				Zoom = fit page
				Rotation = 0
				ScrollPos = -1 -1
				ShowToc = false
			]
		]
		TabIndex = 2
		WindowState = 1
		WindowPos = 789 227 855 793
		SidebarDx = 204
	]
]
```

**Reply by the753q** [reply]
yes it works, thanks!
